receiver/datadogreceiver/translator_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datadogreceiver
+
+import (
+	"bytes"
+	"net/http"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/exportable/pb"
+	"github.com/stretchr/testify/assert"
+	vmsgp "github.com/vmihailenco/msgpack/v4"
+)
+
+func BenchmarkTranslator(b *testing.B) {
```

Comment:
Looks like this file is missing unit tests?

Reply:
Added unite test